vrecord.rb

```diff
@@ -1,29 +1,32 @@
 class Vrecord < Formula
   desc "Capturing a video signal and turning it into a digital file"
   homepage "https://github.com/amiaopensource/vrecord"
-  url "https://github.com/amiaopensource/vrecord/archive/v2020-02-10.tar.gz"
-  version "2020-02-10"
-  sha256 "761aa248856eedc23619d41f5902bba8baba1c67bddc322fcaad7cb241a1edfc"
+  url "https://github.com/amiaopensource/vrecord/archive/v2020-06-26.tar.gz"
+  version "2020-06-26"
+  sha256 "e32b77f8da9ece0fe7ed2a0660fc67a1d6ef48ff8a4692161c57223cdaa3412d"
   head "https://github.com/amiaopensource/vrecord.git"
 
   bottle :unneeded
 
-  depends_on "amiaopensource/amiaos/decklinksdk"
+  depends_on "amiaopensource/amiaos/deckcontrol"
   depends_on "amiaopensource/amiaos/ffmpegdecklink"
   depends_on "amiaopensource/amiaos/gtkdialog"
   depends_on "cowsay"
-  depends_on "freetype"
-  depends_on "gnuplot"
-  depends_on "mediaconch"
-  depends_on "mkvtoolnix"
-  depends_on "mpv"
-  depends_on "qcli"
-  depends_on "sdl"
-  depends_on "xmlstarlet"
+
+  on_macos do
+    depends_on "bash"
```

Comment:
Why is this necessary?

Reply:
because the approach to using this [here document](https://github.com/amiaopensource/vrecord/blob/fb7ec511d37130556c11bc429d63a2ad13c4834e/vrecord#L268) doesn't work with the default bash of some versions of macos that we want to support.

Reply:
Thank you for the explanation! That’s true: the macOS 10.15 comes with a shitty version and I haven’t checked what is in 10.16 beta (according to the developper release) or 11.0 (according to the advertising). I didn’t got an error on this… because on my computers there is the last Bash.

I presume, you need at least 4? Should we not include a test of the minimal requested version?